### PR TITLE
Fix regression in OpenGL DrawImageCommand flags

### PR DESCRIPTION
**Pull request by willox**
This fixes a change from 5f1bc914af159b67e3047524e1e6d70d9b410d2a which caused two flags to be set at the same time. The DrawImage shader can't handle it, so you end up with water looking like:

After this change it returns to looking like:

### CODE REVIEW

**Comment by IntelOrca** [comment]
Could you create an enum for `command.flags` so we don't write `(1 << x)` everywhere?
The same constants should probably also be defined in the `.vert` and `.frag` file.

**Comment by willox** [comment]
Is there no better way to share constants with the shaders?

**Comment by IntelOrca** [comment]
Our GLSL files are not currently pre-processed in anyway, so no.

**Comment by ZehM4tt** [comment]
Ops, good catch.

File: src/openrct2-ui/drawing/engines/opengl/OpenGLDrawingEngine.cpp [hunk 1]
```diff
@@ -809,6 +801,18 @@ void OpenGLDrawingContext::DrawSprite(uint32 image, sint32 x, sint32 y, uint32 t
...
+	if (special) {
+		command.flags |= DrawImageCommand::FLAG_TRANSPARENT_SPECIAL;
+	}
+
+    if (image & IMAGE_TYPE_TRANSPARENT) {
+        command.flags |= DrawImageCommand::FLAG_TRANSPARENT;
+    }
+    else if (image & (IMAGE_TYPE_REMAP_2_PLUS | IMAGE_TYPE_REMAP)) {
+        command.flags |= DrawImageCommand::FLAG_REMAP;
+    }
```
Comment (IntelOrca) [inline src/openrct2-ui/drawing/engines/opengl/OpenGLDrawingEngine.cpp]: Our C++ style rules are to put braces on their own lines.

File: src/openrct2-ui/drawing/engines/opengl/OpenGLDrawingEngine.cpp [hunk 2]
```diff
@@ -809,6 +801,18 @@ void OpenGLDrawingContext::DrawSprite(uint32 image, sint32 x, sint32 y, uint32 t
     command.colour = { 0.0f, 0.0f, 0.0f };
     command.bounds = { left, top, right, bottom };
     command.mask = 0;
+    command.flags = 0;
+
+	if (special) {
+		command.flags |= DrawImageCommand::FLAG_TRANSPARENT_SPECIAL;
+	}
```
Comment (IntelOrca) [inline src/openrct2-ui/drawing/engines/opengl/OpenGLDrawingEngine.cpp]: These lines use tabs rather than spaces.

**Pull request merged**